### PR TITLE
Setup travis builds

### PR DESCRIPTION
**Pull request by stefanbuck**
Setup to be built on Travis. 

@yeoman/owners please enable builds for this repository. Thank you!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![Build Status](https://travis-ci.org/yeoman/yeoman-app.svg)](https://travis-ci.org/yeoman/yeoman-app)
```
Comment (arthurvr) [inline README.md]: Let's do the badge on the same line as the title? ([example](https://github.com/yeoman/generator-angular/blob/master/readme.md))

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+matrix:
+  include:
+    - os: linux
+    - os: osx
+
+sudo: false
```
Comment (sindresorhus) [inline .travis.yml]: This is no longer needed.

File: README.md (hunk 1 above)
Comment (sindresorhus) [inline README.md]: :+1:

**Comment by sindresorhus** [comment]
> @yeoman/owners please enable builds for this repository. Thank you!

Done.

**Comment by stefanbuck** [comment]
:warning:  PLEASE DO NOT MERGE :warning: 

The electron tests are not running on travis!

**Comment by stefanbuck** [comment]
:warning:  **PLEASE DO NOT MERGE** :warning: 

Only the mocha tests are running on travis. I'm struggling a bit with the Jasmine test because of the travis environment. Haven't found a solution yet, any idea?

```
[1850:0719/143545:ERROR:browser_main_loop.cc(173)] Running without the SUID sandbox! See https://code.google.com/p/chromium/wiki/LinuxSUIDSandboxDevelopment for more information on developing with the sandbox on.
[1850:0719/143545:ERROR:browser_main_loop.cc(219)] Gtk: cannot open display: 
[1852:0100/000000:ERROR:zygote_linux.cc(588)] write: Broken pipe
```

**Comment by sindresorhus** [comment]
Electron requires a display. So you need to use a virtual one like `xvfb`: http://docs.travis-ci.com/user/gui-and-headless-browsers/#Using-xvfb-to-Run-Tests-That-Require-GUI-(e.g.-a-Web-browser)

**Comment by stefanbuck** [comment]
Now everything works as expected. Merci @sindresorhus

**Comment by ruyadorno** [comment]
LGTM :+1: @stefanbuck  is it ready to merge now?

**Pull request merged**